source/reference/mongodb-wire-protocol.txt

```diff
@@ -117,6 +117,14 @@ follows:
 Request Opcodes
 ~~~~~~~~~~~~~~~
 
+.. note:: 
+   Starting with MongoDB 2.6 and :data:`~isMaster.maxWireVersion` ``3``, 
+   modern drivers use the :ref:`database commands<collection-commands>` 
```

Comment:
Done. I perused the drivers via ecosystem and none of the driver versions we list lack support for MongoDB 2.6. I'd assume by 'modern' it simply meant anything made within the last few years.